test/Microsoft.ML.Tests/Scenarios/OvaTest.cs

```diff
@@ -133,10 +133,10 @@ public void OvaLinearSvm()
             });
 
             // Data
-            var data = reader.Read(GetDataPath(dataPath));
+            var data = mlContext.Data.Cache(reader.Read(GetDataPath(dataPath)));
 
             // Pipeline
-            var pipeline = new Ova(mlContext, new LinearSvm(mlContext),  useProbabilities: false);
+            var pipeline = new Ova(mlContext, new LinearSvm(mlContext, numIterations: 100),  useProbabilities: false);
```

Comment:
Since the shuffled sequence changed, it now takes more iterations to converge to previous solution.